pyproject.toml

```diff
@@ -0,0 +1,5 @@
+[build-system]
+requires = ["setuptools>=44", "wheel", "setuptools_scm[toml]>=3.4.3"]
```

Comment:
I'm not that familiar with pyproject.toml yet, but what is that `setuptools_scm[toml]>=3.4.3` for? It's not listed at https://packaging.python.org/tutorials/packaging-projects/#creating-pyproject-toml

Reply:
It is needed to fetch version from git tags. The version restrictiin and `toml` extra iss because we configure it via `pyproject.toml`